scripts/ci/pylint_todo.txt

```diff
@@ -1,4 +1,5 @@
 ./airflow/configuration.py
+./airflow/configuration.py:1:0:
```

Comment:
What is the purpose of this change?

Reply:
Odd that I didn't spot that - I ran the pylint todo refresh script, to make sure I hadn't missed anything that might get picked up in Travis, and didn't see any of hat it had added that line :/ my bad, I'll remove it and repush

Reply:
Yeah. the script has this little bug :)? Maybe you can fix it ;).

Reply:
I'll take a look, and open a ticket

Reply:
pylint script ticketed, fixed, PRed and merged :)
This instance has been corrected.

Reply:
Thanks @ratb3rt !